### PR TITLE
Added color support

### PR DESCRIPTION
**Pull request by Bigous**
Added color suport thru cli-color

### CODE REVIEW

**Comment by Bigous** [comment]
Sorry could you cancell this pull?
I commented something for test and forgot to uncomment it :(

**Comment by spion** [comment]
You can `push -f` if you like or push additional commits and it will add them here :)

Will this cause problems with doing e.g. `:read !howdoi sort array in C --code` in vim, or does it strip color codes?

**Comment by Bigous** [comment]
Added an option --no-color to continue compatible with everething... is it better to stay without color as default and add --color when we want a colored respons from the console?

**Comment by spion** [comment]
I personally prefer no-color by default. But if we can make the color unobtrusive in most cases, why not? :P

Another thing I noticed to consider is terminal, are the colors working on both white/black bg?

File: howdoi.js [hunk 1]
```diff
@@ -47,42 +67,43 @@ request(url, function(e, r, body) {
...
     else request(links[args.result - 1], function(e, r, body) {
         var $ = cheerio.load(body);
         var answers = $('.answer .post-text').map(function(i, el) {
             if (args.code) {
-                return $(el).find('pre').text();
+                return aCode($(el).find('pre').text());
             }
-            return $(el).text();
+            var code = $(el).find('pre').text();
+            return $(el).text().replace(code, aCode(code));
```
Comment (spion) [inline howdoi.js]: this is going to be harder because some SO answers have multiple blocks of code

**Comment by Bigous** [comment]
Ok, I'll set the no-color as default and setup color with --color flag so.
cli-color colors the text adding ansi color string to the text, so it should work with any background. I choose the dark green as code color, the dark red as error color, the dark gray as link color and dark white (silver) as the text color... it should work on both, black and white background... but will not work on dark green, dark red, gray or silver background :D

**Comment by Bigous** [comment]
Ok, done ;)
You can try it before accept the pull - i've just tryed it with vim, and it shows the ansi code instead of interpret it as color as expected.
So the best option was to put no-color as default.
And tks to the advice "push -f" I'm very new to git ... I use SVN in all my projects in my company so, with no pratice in git, it's more dificult to learn :dancer: 
[]s

File: howdoi.js (hunk 1 above)
Comment (Bigous) [inline howdoi.js]: Yeap... and the replace too... workin on it...

**Comment by spion** [comment]
cool, i see you wrote you're replacing the code with foreach, I'll merge it after that. You can also make it default to having colors except when passed `--code` - I'd merge that too

**Comment by Bigous** [comment]
Ok, now it's ready :D 

Could you see if it could be at the your repository?

Thanks in advance.

**Comment by spion** [comment]
Merged and published as 0.4.4 :D

**Pull request merged**